src/main/java/com/splunk/ibm/mq/WMQContext.java

```diff
@@ -70,7 +69,7 @@ private void addEnvProperty(Hashtable env, String propName, Object propVal) {
     if (null != propVal) {
       if (propVal instanceof String) {
         String propString = (String) propVal;
-        if (Strings.isNullOrEmpty(propString)) {
+        if (propString.isEmpty()) {
```

Comment:
this reads like a potential semantic change, but the cast would NPE above, so we're already guarded against the null.